packages/grafana-ui/src/components/Select/SelectBase.tsx

```diff
@@ -276,27 +276,39 @@ export function SelectBase<T>({
           IndicatorsContainer(props: any) {
             const { selectProps } = props;
             const { value, showAllSelectedWhenOpen, maxVisibleValues, menuIsOpen } = selectProps;
+            const additionalIndicators = [];
 
             if (maxVisibleValues !== undefined) {
               const selectedValuesCount = value.length;
-              const indicatorChildren = [...props.children];
-              indicatorChildren.splice(
-                -1,
-                0,
+              additionalIndicators.push(
                 renderExtraValuesIndicator({
                   maxVisibleValues,
                   selectedValuesCount,
                   showAllSelectedWhenOpen,
                   menuIsOpen,
                 })
               );
-              return <IndicatorsContainer {...props}>{indicatorChildren}</IndicatorsContainer>;
             }
 
-            return <IndicatorsContainer {...props} />;
+            if (
+              allowCustomValue &&
+              // Currently cannot enter new values if this `isSearchable` false!
+              isSearchable &&
+              (creatableProps.allowCreateWhileLoading || !isLoading)
+            ) {
+              additionalIndicators.push(<Icon name="keyboard" />);
+            }
+
+            const indicatorChildren = [...props.children];
+            // Any additional indicators inserted before last
+            if (indicatorChildren.length > 0) {
+              indicatorChildren.splice(-1, 0, additionalIndicators);
+            }
+
+            return <IndicatorsContainer {...props}>{indicatorChildren}</IndicatorsContainer>;
```

Comment:
what's the intended logic here, show the keyboard icon for any select that you can use the keyboard in? or only for when you can create a custom value?

for example you can search via the keyboard in this select (and most select's tbh) but there's no keyboard icon shown:
![image](https://user-images.githubusercontent.com/20999846/152775961-11660a62-4c05-45d6-a52d-3978793e8d1e.png)